File: src/org/mtransit/android/data/ScheduleProviderProperties.java
package org.mtransit.android.data;

public class ScheduleProviderProperties {

	private String targetAuthority;
	private String authority;

	public ScheduleProviderProperties(String authority, String targetAuthority) {
		this.authority = authority;
		this.targetAuthority = targetAuthority;
	}

	public String getTargetAuthority() {
		return this.targetAuthority;
	}

	public String getAuthority() {
		return authority;
	}

	@Override
	public String toString() {
		return new StringBuilder().append(ScheduleProviderProperties.class.getSimpleName()).append('{') //
				.append("authority:").append(authority).append(',') //
				.append("targetAuthority:").append(targetAuthority).append(',') //
				.append('}').toString();
	}

}

File: src/main/java/org/mtransit/android/data/ScheduleProviderProperties.kt
package org.mtransit.android.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(
    tableName = "schedule_provider_properties",
)
data class ScheduleProviderProperties(
    @PrimaryKey
    @ColumnInfo(name = "authority")
    override val authority: String,
    @ColumnInfo(name = "target_authority")
    override val targetAuthority: String,
    @ColumnInfo(name = "pkg")
    override val pkg: String,
) : ITargetedProviderProperties